cinch/roles/nginx/tasks/selinux.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: enable selinux overrides
+  seboolean:
+    name: "{{ item }}"
+    state: yes
```

Comment:
"yes" Is used here whereas "true/false" is used throughout the rest of this role and Cinch in general.